romancal/tweakreg/tweakreg_step.py

```diff
@@ -0,0 +1,560 @@
+                    f"Skipping alignment to the {self.abs_refcat} reference "
+                    "catalog!"
+                )
+            else:
+                # align images:
+                # Update to separation needed to prevent confusion of sources
+                # from overlapping images where centering is not consistent or
+                # for the possibility that errors still exist in relative overlap.
+                xyxymatch_gaia = XYXYMatch(
+                    searchrad=self.abs_searchrad,
+                    separation=self.abs_separation,
+                    use2dhist=self.abs_use2dhist,
+                    tolerance=self.abs_tolerance,
+                    xoffset=self.abs_xoffset,
+                    yoffset=self.abs_yoffset
+                )
+
+                # Set group_id to same value so all get fit as one observation
+                # The assigned value, 987654, has been hard-coded to make it
+                # easy to recognize when alignment to GAIA was being performed
+                # as opposed to the group_id values used for relative alignment
+                # earlier in this step.
+                for imcat in imcats:
+                    imcat.meta['group_id'] = 987654
+                    if ('fit_info' in imcat.meta and
+                            'REFERENCE' in imcat.meta['fit_info']['status']):
+                        del imcat.meta['fit_info']
+
+                # Perform fit
+                align_wcs(
+                    imcats,
+                    refcat=ref_cat,
+                    enforce_user_order=True,
+                    expand_refcat=False,
+                    minobj=self.abs_minobj,
+                    match=xyxymatch_gaia,
+                    fitgeom=self.abs_fitgeometry,
+                    nclip=self.abs_nclip,
+                    sigma=(self.abs_sigma, 'rmse'),
+                    ref_tpwcs=imcats[0]
+                )
+
+        for imcat in imcats:
+            image_model = imcat.meta['image_model']
+            image_model.meta.cal_step["tweakreg"] = 'COMPLETE'
+
+            # retrieve fit status and update wcs if fit is successful:
+            if 'SUCCESS' in imcat.meta.get('fit_info')['status']:
+
+                # Update/create the WCS .name attribute with information
+                # on this astrometric fit as the only record that it was
+                # successful:
+                if ALIGN_TO_ABS_REFCAT:
+                    # NOTE: This .name attrib agreed upon by the JWST Cal
+                    #       Working Group.
+                    #       Current value is merely a place-holder based
+                    #       on HST conventions. This value should also be
+                    #       translated to the FITS WCSNAME keyword
+                    #       IF that is what gets recorded in the archive
+                    #       for end-user searches.
+                    imcat.wcs.name = f"FIT-LVL3-{self.abs_refcat}"
```

Comment:
Should this name be changed to `f"FIT-LVL2-{self.abs_refcat}"` because tweakreg is running in the Level 2 stage of the Roman pipeline?